### ISSUE TITLE
Songs are different

### ISSUE THREAD

**Issue by hellomyzn** [SwapnilSoni1999/spotify-dl#159]
**Describe the bug**
I tried to download [this album](https://open.spotify.com/album/4TPmFi8XmAESJ8JYeKJElU). but when I check the songs downloaded, Some songs are different.

**To Reproduce**
1. Run the command below
```
$  docker run -it -v ~/Downloads/spotify:/download spotify-dl https://open.spotify.com/album/4TPmFi8XmAESJ8JYeKJElU
````
2. Wait for done
3. Check the songs downloaded and on Spotify

**Expected behavior**
Songs are the same as on Spotify

**Screenshots**
After running the command(Step 1)
<img width="1298" alt="image" src="https://user-images.githubusercontent.com/20104403/168205605-958d1bfb-c623-44fa-8e82-667360264d9e.png">

**Desktop (please complete the following information):**
<img width="519" alt="image" src="https://user-images.githubusercontent.com/20104403/168205008-8285a913-10da-4fca-8c21-48c85c486721.png">

 - Browser: chrome
 - Version: spotify-dl:latest

**Additional context**
I'm just wondering if Spotify-dl downloads the songs from youtube because as you can see the pic below, you can see `searching youtube with keywords "DOLLY - Blue"`. 
Just in case, I searched the keyowrd (DOLLY - Blue) on Youtube and then I checked the [video](https://www.youtube.com/watch?v=CBIYAVCVrEM). 
Then the audio is the same as the song downloaded through spotify-dl
<img width="1294" alt="image" src="https://user-images.githubusercontent.com/20104403/168205728-8b2ed900-d50a-41f4-9c1c-96ec965dedea.png">

Thanks

**Comment by phyzical** [comment]
yeah thats correct, what we do is use the spotify api to get a decent amount of context then hit youtube with this context and we have to just "guess" which ones correct its pretty "dumb", then rip the first relevant youtube link found.

tldr what you've said is exactly how it works.

**Comment by hellomyzn** [comment]
@phyzical 

Thank you for your explanation.
I got it 👍
